roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_priorities.py

```diff
@@ -91,7 +91,7 @@ def run(self, terms, variables=None, zones_enabled=True, short_version=None,
                 {'name': 'InterPodAffinityPriority', 'weight': 1}
             ])
 
-        if short_version in ['3.5', '3.6']:
+        if short_version in ['3.5', '3.6', '3.7']:
```

Comment:
@sdodson The only new default predicate I see in 3.7, that was not in 3.6 is  NoVolumeNodeConflict. But no change is required in default configuration.